js/client-teambuilder.js

```diff
@@ -3135,6 +3138,8 @@
 
 			if (this.curTeam && this.ignoreEVLimits) {
 				evs = {hp:252, atk:252, def:252, spa:252, spd:252, spe:252};
+				if (!moveCount['PhysicalAttack']) delete evs.atk;
+				if (!moveCount['SpecialAttack']) delete evs.spa;
```

Comment:
It's not strictly necessary for what I put it there to do (stop the builder from assuming that all bh sets are mixed), but I put it here instead of where the minusStat is determined because Power Split is a thing, if a very rare one.